src/components/customers/addDrawer/ExternalAppsAccordion/index.tsx

```diff
@@ -0,0 +1,252 @@
+import { gql } from '@apollo/client'
+import { FormikProps } from 'formik'
+import { useEffect, useState } from 'react'
+
+import { Accordion, Alert, Button, Popper, Typography } from '~/components/designSystem'
+import {
+  ADD_CUSTOMER_ACCOUNTING_PROVIDER_ACCORDION,
+  ADD_CUSTOMER_CRM_PROVIDER_ACCORDION,
+  ADD_CUSTOMER_PAYMENT_PROVIDER_ACCORDION,
+  ADD_CUSTOMER_TAX_PROVIDER_ACCORDION,
+  MUI_BUTTON_BASE_ROOT_CLASSNAME,
+} from '~/core/constants/form'
+import { INTEGRATIONS_ROUTE } from '~/core/router'
+import {
+  CreateCustomerInput,
+  IntegrationTypeEnum,
+  ProviderTypeEnum,
+  UpdateCustomerInput,
+} from '~/generated/graphql'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import PSPIcons from '~/public/images/psp-icons.svg'
+import { MenuPopper } from '~/styles'
+
+import { AccountingProvidersAccordion } from './AccountingProvidersAccordion'
+import { CRMProvidersAccordion } from './CRMProvidersAccordion'
+import { PaymentProvidersAccordion } from './PaymentProvidersAccordion'
+import { TaxProvidersAccordion } from './TaxProvidersAccordion'
+import { scrollToIntegration } from './utils'
+
+gql`
+  fragment CustomerForExternalAppsAccordion on Customer {
+    id
+    customerType
+    # Name in the customer is netsuiteCustomer, but it's used as integrationCustomer in the create update inputs
+    netsuiteCustomer {
```

Comment:
I don't see any `xxxCustomer` object being used in this file.
Maybe we should defined them as fragment in the dedicated files and import them here instead?